packages/manager/modules/pci/src/projects/project/translations/Messages_fr_FR.json

```diff
@@ -12,9 +12,10 @@
   "pci_projects_project_activate_private_networks": "Gérer les réseaux privés (vRack)",
   "pci_projects_project_buy_ip_failover": "Gérer les IPs Failover",
   "pci_projects_project_is_new": "Nouveau",
-  "pci_projects_project_discovery_banner_title_1": "Activez votre projet en ajoutant un moyen de paiement et profitez de",
-  "pci_projects_project_discovery_banner_title_2": "200€ de crédit offert.",
-  "pci_projects_project_discovery_banner_description": "Vous êtes actuellement en mode découverte. En activant votre projet, vous pourrez démarrer vos ressources cloud en quelques minutes. ",
+  "pci_projects_project_discovery_banner_promotion_title_1": "Activez votre projet en ajoutant un moyen de paiement et profitez de",
+  "pci_projects_project_discovery_banner_promotion_title_2": " {{ amount }} de crédit offert.",
```

Comment:
Can you make this a unique sentence with a unique translation key ?

Reply:
This translation key needs to be separated because the second part has a different typography when displayed.